test/util/nightwatch-commands/fill.js

```diff
@@ -2,10 +2,15 @@
  * Clears the current value and if |value| is specified,
  *  enters the |value| in.
  */
-exports.command = function fill(selector, value) {
+exports.command = function fill(selector, value, callback) {
   this.clearValue(selector);
   if (typeof value !== 'undefined') {
-    this.setValue(selector, value);
+    // When callback is passed as undefined, setValue assumes selector is the locating strategy
```

Comment:
Yeah, it should work just fine according to the docs: http://nightwatchjs.org/api/#setValue
Unfortunately, it doesn't 🙁 It was throwing an error telling me that the location strategy of `input[name="whatever"]` is invalid and it must be [a whole list of things including `css selector`].

I would have thought that not passing the third param would act the same as passing the third param as undefined, but...I guess not so much.

The callback isn't used, but I wrote this bit while testing a way to fill either one element or another, but...that was just awful, so I abandoned it.